src/ModuleTestLite.cs

```diff
@@ -303,7 +304,7 @@ private void updateCore()
 			if (Core.Instance == null)
 				return;
 			if (!Core.Instance.du.ContainsKey(configuration)) /* should never happen */
-				Core.Instance.du[configuration] = 0d;
+				Core.Instance.du[configuration] = startingDu;
```

Comment:
Not sure if it's worth it but you could support upgrading existing games by feeding the startingDu into Math.Max() unconditionally rather than using it in place of 0d in the various conditionals.